ferrocene/doc/qualification-plan/src/standards/iec-61508.rst

```diff
@@ -8,127 +8,169 @@ IEC 61508 requirements
    :header-rows: 1
 
    * - Clause
-     - Document chapter(s)
+     - ``rustc``
+     - ``rustfmt``
 
    * - 7.4.4.1
      - N/A
+     - N/A
 
    * - 7.4.4.2
      - N/A
+     - N/A
 
    * - 7.4.4.3
      - This qualification is the justification
+     - This qualification is the justification
 
    * - 7.4.4.4
-     - | :doc:`Safety Manual - Usage <safety-manual:rustc/usage>`
-       | :doc:`Safety Manual - Degraded environment <safety-manual:rustc/degraded-environment>`
-       | :doc:`Evaluation Report - Use cases <evaluation-report:rustc/use-cases>`
+     - | :doc:`Safety Manual - rustc - Usage <safety-manual:rustc/usage>`
+       | :doc:`Safety Manual - rustc - Degraded environment <safety-manual:rustc/degraded-environment>`
+       | :doc:`Evaluation Report - rustc - Use cases <evaluation-report:rustc/use-cases>`
+     - | :doc:`Safety Manual - rustfmt - Usage <safety-manual:rustfmt/usage>`
+       | :doc:`Evaluation Report - rustfmt - Use cases <evaluation-report:rustfmt/use-cases>`
 
    * - 7.4.4.5
-     - :doc:`Evaluation Report - Tool analysis <evaluation-report:rustc/tool-analysis>`
+     - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
+     - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
 
    * - 7.4.4.6
-     - | :doc:`Evaluation Report - Tool analysis <evaluation-report:rustc/tool-analysis>`
-       | :doc:`Qualification Report - Test results <qualification-report:rustc/index>`
+     - | :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
+       | :doc:`Qualification Report - rustc - Test results <qualification-report:rustc/index>`
+     - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
```

Comment:
This is not evidence that the tool conforms to its specification or documentation. We don't really have it, and we should instead link to the paragraph that says we are not doing the requirements and traceability since all possible failures can be detected.

Reply:
+1 on that comment

Reply:
> we should instead link to the paragraph that says we are not doing the requirements and traceability since all possible failures can be detected.

That is the "IEC 61508 Tool Classification" which is in the Tool analysis. I will link to the subsection.